experimenter/experimenter/nimbus_ui_new/forms.py

```diff
@@ -106,6 +106,109 @@ def clean(self):
         return cleaned_data
 
 
+class NimbusExperimentCloneFormMixin:
+    def __init__(self, *args, parent_slug=None, branch_slug=None, **kwargs):
+        self.parent_slug = parent_slug
+        self.branch_slug = branch_slug
```

Comment:
Is there a reason you did this this way instead of just making these form fields that are passed in through the actual form data?